yt/frontends/amrex/data_structures.py

```diff
@@ -1344,14 +1345,16 @@ def __init__(self, ds, dataset_type="quokka_native"):
             is_checkpoint = True
 
             self._read_particles(
-                "What", # RJ: Quokka uses What for particles, Tracer for Castro, DM for Nyx
+                "Rad_particles", # RJ: Quokka uses Rad_particles? for particles, just a placeholder here
```

Comment:
Instead of defaulting "Rad_particles", can you change it to loop through a list of possible particle types? The possible particle types are `["Rad_particles", "CIC_particles", "tracer_particles"]`. More particle types will be added in the future. If none of those types are found, throw an warning.